plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
But I guess test case should clean after itself if possible - I will look for a way of adding this clean up to this test (not MTR)